tests/python/frontend/tflite/test_forward.py

```diff
@@ -0,0 +1,374 @@
+# pylint: disable=import-self, invalid-name, unused-argument
+"""
+TFLite testcases
+====================
+This article is a test script to test TFLite operator with Relay.
+"""
+from __future__ import print_function
+import numpy as np
+import tvm
+from tvm import relay
+from tvm.contrib import util
+import tensorflow as tf
+from tensorflow.python.framework import constant_op
+from tensorflow.python.ops import nn_ops
+from tensorflow.python.ops import array_ops
+from tensorflow.python.ops import variables
+from tensorflow.contrib.lite.python import interpreter as interpreter_wrapper
+
+import nnvm.testing.tf
+
+
+
+#######################################################################
+# Generic run functions for TVM & TFLite
+# ------------------------------------------
```

Comment:
Doc style ..

Reply:
It is based on `tests/python/frontend/tensorflow/test_forward.py` and just modify `tensorflow` to `Tflite`. Could you explain more about doc style? Thanks.

Reply:
:) sorry, it's the underline again.

Reply:
So, you suggest making `underline(_)` into `number sign(#)`?

Reply:
ping ping @srkreddy1238

Reply:
It was underline(_) length. I see this is already handled.